src/OgRolesHandler.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Contains /Drupal/og/OgRolesHandler
+ */
+
+namespace Drupal\og;
+
+use Drupal\Core\Session\AccountInterface;
+use Drupal\og\Entity\OgRole;
+use Drupal\og\Entity\OgUsersRole;
+
+class OgRolesHandler {
```

Comment:
That classname sounds quite generic, any idea for a better name, maybe OgRolesStorage?